src/engraving/rendering/dev/chordlayout.cpp

```diff
@@ -662,16 +662,6 @@ void ChordLayout::layoutSpanners(Chord* item, System* system, const Fraction& st
 {
     //! REVIEW Needs explanation
     for (const Note* note : item->notes()) {
-        Tie* t = note->tieFor();
-        if (t) {
-            SlurTieLayout::tieLayoutFor(t, system);
-        }
-        t = note->tieBack();
-        if (t) {
-            if (t->startNote()->tick() < stick) {
-                SlurTieLayout::tieLayoutBack(t, system);
-            }
-        }
```

Comment:
Now the `system` and `stick` parameters are unused and can be removed. (And that will require giving the two overloads of `ChordLayout::layoutSpanners` different names, which seems a good idea anyway.)

Actually, why did you decide to move this code out of this method?

Reply:
Cause I meant to delete it (I did now)